build/dsc/templates/dsc_type.rb.erb

```diff
@@ -1,4 +1,5 @@
 require 'pathname'
+require File.expand_path(File.join(File.dirname(__FILE__), '../../puppet_x/puppetlabs/dsc_type_helpers'))
```

Comment:
Also thought I was using standard module mechanism for `require` there, but looking at a few different projects, I see now that we don't really have a standard ;0

It's a hodgepdge of `File.expand_path`, `Pathname.new(__FILE__).dirname` and `require_relative`.

I actually think that `require_relative` is the best way of doing this that I know of - but maybe your firsthand experience says otherwise?